src/global.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
I thought we can use index.css instead of global.css. I was a little confused as most of the time when defining global files in React especially for CSS, many developers make use of index.css

Reply:
I like to call it `global`, I find it to have more meaning than `index`.  Index is more related to `start` or `first` while global is related to `all` or `shared`. But I have changed it to `index.css` to avoid the confusion.